From 18d73fbb1617dfb700bc136304bc2e25b2870825 Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Thu, 26 Sep 2013 17:57:18 +0800
Subject: [PATCH 0297/1072] ENGR00281446 ARM: dts: imx6: correct the power
 supply node name for epdc

git://git.freescale.com/imx/linux-2.6-imx.git imx_3.10.17_1.0.0_beta
commit 982d244c9eb32eaefb40225314661aea487d87cb

The naming convention for power supply node in DTS is "%s-supply".
With this patch regulator_get() will process in the DT way rather than
traditional way. This patch has no functional impact.

Signed-off-by: Robby Cai <R63905@freescale.com>
Signed-off-by: Hongbo Zhong <hongbo.zhong@windriver.com>
---
 arch/arm/boot/dts/imx6dl-sabresd.dts |    6 +++---
 arch/arm/boot/dts/imx6sl-evk.dts     |    6 +++---
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm/boot/dts/imx6dl-sabresd.dts b/arch/arm/boot/dts/imx6dl-sabresd.dts
index 224b328..5713c71 100644
--- a/arch/arm/boot/dts/imx6dl-sabresd.dts
+++ b/arch/arm/boot/dts/imx6dl-sabresd.dts
@@ -116,9 +116,9 @@
 &epdc {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_epdc_0>;
-	V3P3_supply = <&V3P3_reg>;
-	VCOM_supply = <&VCOM_reg>;
-	DISPLAY_supply = <&DISPLAY_reg>;
+	V3P3-supply = <&V3P3_reg>;
+	VCOM-supply = <&VCOM_reg>;
+	DISPLAY-supply = <&DISPLAY_reg>;
 	status = "okay";
 };
 
diff --git a/arch/arm/boot/dts/imx6sl-evk.dts b/arch/arm/boot/dts/imx6sl-evk.dts
index 3ed918f..f593a3a 100644
--- a/arch/arm/boot/dts/imx6sl-evk.dts
+++ b/arch/arm/boot/dts/imx6sl-evk.dts
@@ -140,9 +140,9 @@
 &epdc {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_epdc_0>;
-	V3P3_supply = <&V3P3_reg>;
-	VCOM_supply = <&VCOM_reg>;
-	DISPLAY_supply = <&DISPLAY_reg>;
+	V3P3-supply = <&V3P3_reg>;
+	VCOM-supply = <&VCOM_reg>;
+	DISPLAY-supply = <&DISPLAY_reg>;
 	status = "okay";
 };
 
-- 
1.7.5.4

